ci-templates/github/model-complexity-badge/action.yml

```diff
@@ -0,0 +1,72 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capellambse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Generate Model Complexity Badge'
+description: 'An action to create and push a model complexity badge'
+
+inputs:
+  commit_msg:
+    description: 'The commit message for updating the model complexity badge'
+    required: false
+    default: "docs: Update model complexity badge"
+  output_file:
+    description: 'The file path to store the generated model badge'
+    required: false
+    default: model-complexity-badge.svg
+  entry_point:
+    description: 'The entry point of the used model'
```

Comment:
The `entry_point` can actually also be made optional, by leveraging the autodetection logic introduced in v0.5.29. In the absence of a user-provided entrypoint, we can simply pass `"."` to the `MelodyModel()` constructor or `loadcli()` call (this could just be set as the default here).